app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -87,9 +88,13 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
     override fun onStart() {
         super.onStart()
         // https://github.com/mozilla-mobile/fenix/issues/14279
```

Comment:
^^ This link would be a leftover here, but I think we need to take a look at the profiler first, because that's not something we should regress. Resizing the (Gecko)view will significantly slow down subsequent page loads.

To verify we'd need to e.g.
- Open wikipedia.org
- Open search
- Open mozilla.org

and make sure we're not causing two (unnecessary) resizes here when opening search and when closing it to navigate to the new page. Happy to pair / help with profiling this, let me know :).

Reply:
I'd love to pair on the profiling! (it's my first time 😱)